java/protoc/pom.xml

```diff
@@ -30,4 +30,115 @@
       scm:git:https://github.com/protocolbuffers/protobuf.git
     </connection>
   </scm>
+  <build>
```

Comment:
This all shouldn't be added. I changed the way the pom file works so we upload one at a time and don't list the artifacts like this.